lab_08/lib.h

```diff
@@ -0,0 +1,30 @@
+
+#include <stdio.h>
+#include <time.h>
+#include <stdlib.h>
+
+#define SIZE 8
```

Comment:
This defies logic. Using a predefined size for such a task limits the applicability of your functions (or library).